test/suites/vm.sh

```diff
@@ -24,9 +24,10 @@ test_vm_empty() {
   echo "Too small VMs"
   ! lxc launch --vm --empty v1 -c limits.memory=0 -d "${SMALL_ROOT_DISK}" || false
   ! lxc launch --vm --empty v1 -c limits.memory=0% -d "${SMALL_ROOT_DISK}" || false
-  # VMs don't support limits.memory in % but it's only detect at start time so needs cleanup
-  ! lxc launch --vm --empty v1 -c limits.memory=10% -d "${SMALL_ROOT_DISK}" || false
-  lxc delete v1
+
+  echo "Tiny VM with tiny memory"
+  lxc launch --vm --empty v1 -c limits.memory=1% -d "${SMALL_ROOT_DISK}"
+  lxc delete -f v1
```

Comment:
Thanks for also adapting the tests. BTW, when I first ran into this, I had the working `lxc launch` on line 46 use a `%` limit.

The reason to tweak an existing one is because init/launch is relatively expensive so it's best to limit/combine them when possible.